tests/unittest_data/demo.csv

```diff
@@ -0,0 +1,10 @@
+demo,Name,gender,demo two,demo num,demo_cat,target
```

Comment:
`test_data` might be a better name for this folder as we will have system tests to that use this data?